src/parser/postgresparser.cpp

```diff
@@ -194,7 +194,10 @@ parser::TableRef* PostgresParser::FromTransform(List* root) {
               RangeSubselectTransform(reinterpret_cast<RangeSubselect*>(node)));
           break;
         }
-        default: { LOG_ERROR("From Type %d not supported yet...", node->type); }
+        default: {
+          LOG_ERROR("From Type %d not supported yet...", node->type);
+          throw NotImplementedException(" ");
```

Comment:
The error message is printed using the LOG_ERROR instead since we want to show the type of the node that is causing the error, formatting is not supported in the Exception message.